src/main/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetDomain.scala

```diff
@@ -0,0 +1,40 @@
+package io.iohk.ethereum.faucet.jsonrpc
+
+import akka.util.ByteString
+import io.iohk.ethereum.domain.Address
+import io.iohk.ethereum.faucet.FaucetStatus
+import io.iohk.ethereum.jsonrpc.JsonMethodsImplicits
+import io.iohk.ethereum.jsonrpc.JsonRpcError.InvalidParams
+import io.iohk.ethereum.jsonrpc.serialization.JsonMethodDecoder.NoParamsMethodDecoder
+import io.iohk.ethereum.jsonrpc.serialization.{JsonEncoder, JsonMethodDecoder}
+import org.json4s.JsonAST.{JArray, JObject, JString}
+
+object FaucetDomain {
+
+  case class SendFundsRequest(address: Address)
+  object SendFundsRequest extends JsonMethodsImplicits {
```

Comment:
I prefer to make `FaucetMethodsImplicits` a separate trait. Now we are mixing the domain model with serialization layer

Reply:
Done